AssemblyOne/AssemblyOne/Employee.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssemblyOne
+{
+    public class Employee
+    {
+        public const int numPublic = 55;
```

Comment:
all constants (const) should be start with Upper case letter, for example

numPublic => `NumPublic`